recipe/recipe.yaml

```diff
@@ -34,6 +34,11 @@ requirements:
     - ${{ compiler("cxx") }} # needed to compile libduckdb-sys
     - ${{ compiler("rust") }}
     - cargo-bundle-licenses
+    - if: build_platform != target_platform
+      then:
+        - cross-python_${{ target_platform }}
+        - maturin >=1.7,<2.0
```

Comment:
As far as I can tell, the only difference between this and https://github.com/conda-forge/stacrs-feedstock/pull/4/files#diff-ad6aed43f0cf479348ccccec8e7185c4b70582192329f11908dd22a731771702, is the addition of `maturin` here :shrug: So you were super close @gadomski!